docs/publishing-to-rubygems.md

```diff
@@ -0,0 +1,29 @@
+# Publishing to RubyGems
+
+1. Checkout **master** and pull latest changes.
+
+2. Create and checkout a new branch (`release-[version-number]`).
+  The version number is determined by looking at the [current "Unreleased" changes in CHANGELOG](../CHANGELOG.md) and updating the previous release number depending on the kind of entries:
+
+  - `Breaking changes` corresponds to a `major` (1.X.X) change.
+  - `New features` corresponds to a `minor` (X.1.X) change.
+  - `Fixes` corresponds to a `patch` (X.X.1) change.
+
+  For example if the previous version is `2.3.0` and there are entries for `Breaking changes` then the new release should be `3.0.0`.
+
+  See the [Semantic Versioning](https://semver.org/) for more information about versioning.
+
+3. Update [`CHANGELOG.md`](../CHANGELOG.md) "Unreleased" heading with the new version number.
+
+4. Update [`lib/govuk_publishing_components/version.rb`](../lib/govuk_publishing_components/version.rb) version with the new version number.
+
+5. Run `bundle install && npm install` to ensure you have the latest dependencies installed.
+
+6. Commit changes. These should include updates in the following files:
+  - [`CHANGELOG.md`](../CHANGELOG.md)
+  - [`lib/govuk_publishing_components/version.rb`](../lib/govuk_publishing_components/version.rb)
+  - [`Gemfile.lock`](../Gemfile.lock)
+
+7. Create a pull request and copy the changelog text for the current version.
```

Comment:
"Copy the changelog text into the PR description"? Otherwise it's not clear what you're copying it to.